static/data/community.ts

```diff
@@ -2,7 +2,7 @@ import { MEETING_URL } from './global';
 const header = {
   title: 'Community',
   subtitle:
-    'We want your feedback, issues, patches, and involvement in the development of Podman. **Chat** with us on Slack, IRC, or on our **mailing list**. Submit **issues & pull requests** (see our [CONTRIBUTING guide](https://github.com/containers/podman/blob/main/CONTRIBUTING.md) on how.) Participate in one of our twice-monthly community meetings. You are welcome in our community!',
+    'We want your feedback, issues, patches, and involvement in the development of Podman. **Chat** with us on Matrix, Discord, IRC, or on our **mailing list**. Submit **issues & pull requests** (see our [CONTRIBUTING guide](https://github.com/containers/podman/blob/main/CONTRIBUTING.md) on how.) Participate in one of our twice-monthly community meetings. You are welcome in our community!',
```

Comment:
Do we have CNCF approval for Matrix, Discord and IRC ? If it's only matrix, then maybe we should skip Discord and IRC mentions from here. Not a strong opinion.